.github/workflows/release.yml

```diff
@@ -37,60 +37,63 @@ jobs:
           path: cdn
 
       - name: Prepare version number
+        id: prepare
         working-directory: app-frontend
         run: |
           echo Clone, copy, commit and push to Altinn-CDN
           CURRENT_VERSION=`git describe --abbrev=0 --tags 2>/dev/null`
           CURRENT_VERSION_PARTS=(${CURRENT_VERSION//./ })
-          APP_MAJOR=${CURRENT_VERSION_PARTS[0]:1}
-          APP_MINOR=${CURRENT_VERSION_PARTS[1]}
-          APP_PATCH=${CURRENT_VERSION_PARTS[2]}
-          echo altinn-app-frontend version: $APP_MAJOR.$APP_MINOR.$APP_PATCH
-          AUTHOR_FULL=$(git log -1 | grep Author)
-          AUTHOR_NAME=$(git log -1 | grep Author | cut -d' ' -f2)
-          AUTHOR_EMAIL=$(git log -1 | grep Author | cut -d' ' -f3 | cut -d'<' -f2 | cut -d'>' -f1)
-          COMMIT_ID=$(git rev-parse HEAD~0)
+          echo ::set-output name=APP_MAJOR::${CURRENT_VERSION_PARTS[0]:1}
+          echo ::set-output name=APP_MINOR::${CURRENT_VERSION_PARTS[1]}
+          echo ::set-output name=APP_PATCH::${CURRENT_VERSION_PARTS[2]}
```

Comment:
Consider if this step should have an output ala `APP_FULL_VERSION` since we have many uses where we combine alle of these. This would in my opinion make the rest of the workflow more readable.
```
echo ::set-output name=APP_FULL_VERSION::${APP_MAJOR}.${APP_MINOR}.${APP_PATHC}
```